### PR TITLE
Update Codespaces information

### PR DESCRIPTION
**Pull request by StevenMaude**
See opensafely-core/codespaces-initiative#38.

### CODE REVIEW

**Comment by StevenMaude** [comment]
Again, as in #1485, I've broken out potentially useful sections to their own pages.

I'm not particularly tied to keeping them, but it makes it easy to decide whether they are still useful.

File: docs/getting-started/how-to/use-github-codespaces-in-your-project/index.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+   It may take a few minutes before the codespace loads
+   and displays the Visual Studio Code editor.
+
+!!! info
+    For "Create codespace on main",
+    "main" refers to the Git branch to use in the codespace.
+
+    If you are working on another repository branch,
+    it is possible to select "New from options",
+    which allows you to choose the branch.
```
Comment (StevenMaude) [inline docs/getting-started/how-to/use-github-codespaces-in-your-project/index.md:26]: thought: I'm not sure if we want to mention branches at all in this page. There's a mention here, and further down when talking about pushing changes to the GitHub remote.

File: docs/getting-started/how-to/use-github-codespaces-in-your-project/index.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+Deleting a codespace removes the codespace entirely,
+preventing any further CPU or storage quota usage by the codespace.
+
+!!! info
+    If you have changes in the codespace that have not been published to the repository,
+    GitHub will warn you to confirm that you are sure you want to delete the codespace.
+
+## Developing OpenSAFELY projects in GitHub Codespaces
+
+### How to ensure your work is saved
```
Comment (iaindillingham) [inline docs/getting-started/how-to/use-github-codespaces-in-your-project/index.md:71]: How would you feel about rephrasing this in the active voice?

```suggestion
### How save your work
```

**Review by iaindillingham — APPROVED** [review]
Couple of comments, but no deal-breakers. Thanks for such a thorough job!

File: docs/getting-started/how-to/create-a-code-repository-for-your-project/index.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+## Create a code repository for your project
```
[the remark this answers is not in the document]
Comment (StevenMaude) [reply docs/getting-started/how-to/create-a-code-repository-for-your-project/index.md]: I've fixed this, and the other heading issue, by adding the pages to the table of contents.

**Pull request merged**